lib/paranoia/paranoia.c

```diff
@@ -325,49 +317,42 @@ i_paranoia_overlap2(int16_t *buffA,int16_t *buffB,
  * (I.e., offset indicates the distance between what A considers sample N
  * on the CD and what B considers sample N.)
  */
-static inline long int
-do_const_sync(c_block_t *A,
-	      sort_info_t *B,
-	      unsigned char *flagB,
-	      long posA, long posB,
-	      long *begin, long *end, long *offset)
-{
-  unsigned char *flagA=A->flags;
-  long ret=0;
+static inline long int do_const_sync(c_block_t *A, sort_info_t *B,
+                                     unsigned char *flagB, long posA, long posB,
+                                     long *begin, long *end, long *offset) {
+  unsigned char *flagA = A->flags;
+  long ret = 0;
 
   /* If we're doing any verification whatsoever, we have flags in stage
    * 1, and will take them into account.  Otherwise (e.g. in stage 2),
    * we just do the simple equality test for samples on both sides of
    * the initial match.
    */
-  if (flagB==NULL)
-    ret=i_paranoia_overlap(cv(A), iv(B), posA, posB,
-			   cs(A), is(B), begin, end);
-  else
-    if ((flagB[posB]&FLAGS_UNREAD)==0)
-      ret=i_paranoia_overlap2(cv(A), iv(B), flagA, flagB,
-			      posA, posB, cs(A), is(B),
-			      begin, end);
+  if (flagB == NULL)
+    ret =
+        i_paranoia_overlap(cv(A), iv(B), posA, posB, cs(A), is(B), begin, end);
+  else if ((flagB[posB] & FLAGS_UNREAD) == 0)
+    ret = i_paranoia_overlap2(cv(A), iv(B), flagA, flagB, posA, posB, cs(A),
+                              is(B), begin, end);
 
   /* Small matching runs could just be coincidental.  We only consider this
    * a real match if it's long enough.
    */
   if (ret > MIN_WORDS_SEARCH) {
-    *offset=+(posA+cb(A))-(posB+ib(B));
+    *offset = +(posA + cb(A)) - (posB + ib(B));
```

Comment:
I'm surprised by the `+` between `=` and `(`: either it is normal and then the `+` is useless (I guess?) or it should be `+=`

Reply:
I've traced this change back to the SVN change from r36 to r37 which has this generic commit

```
------------------------------------------------------------------------

r37 | xiphmont | 1999-08-15 18:29:59 -0400 (Sun, 15 Aug 1999) | 6 lines



Imported cdparanoia III alpha 9.3 into CVS
```

The 9.2 line  was: 

```python
    *offset=(posA+isort_begin(A))-(posB+isort_begin(B));
```

Given the lack of information about the motivation for the "+" I'll remove it so it does not cause others to wonder about it.